pkg/wasm/utils.go

```diff
@@ -30,10 +31,33 @@ func AuthServiceTimeout() string {
 	return env.GetString("AUTH_SERVICE_TIMEOUT", "200ms")
 }
 
+func AuthServiceFailureMode() FailureModeType {
+	return parseFailureModeValue("AUTH_SERVICE_FAILURE_MODE", FailureModeDeny)
+}
+
 func RatelimitServiceTimeout() string {
 	return env.GetString("RATELIMIT_SERVICE_TIMEOUT", "100ms")
 }
 
+func RatelimitServiceFailureMode() FailureModeType {
+	return parseFailureModeValue("RATELIMIT_SERVICE_FAILURE_MODE", FailureModeAllow)
+}
+
+func parseFailureModeValue(envVarName string, defaultValue FailureModeType) FailureModeType {
+	value := os.Getenv(envVarName)
+	if value == "" {
+		return defaultValue
+	}
+
+	switch value {
+	case string(FailureModeAllow), string(FailureModeDeny):
+		return FailureModeType(value)
+	default:
+		fmt.Printf("Warning: Invalid value '%s' for %s. Using default value '%s'.\n", value, envVarName, defaultValue)
```

Comment:
Seems slightly odd to use standard printing here instead of using a logger but I guess that's because it's not easy to get the logger here?

Reply:
Yup, I wanted to avoid having to create an instance of the logger just for this... but I could if needed.

Reply:
I could also remove the message xD

Reply:
I think it's nice to have the message at least to know that if you set it with a typo there's a reason why it's not being set.. but it's also going to look weird with interleaved `Printf` messages with the operator logs every time there's an event that triggers modification to the plugin that goes through this code. I don't want to hold up this PR though not a big issue

Reply:
I would panic, it is wrong to have an invalid value

Reply:
But happy to merge this as it is as well

Reply:
Take a look at https://github.com/Kuadrant/kuadrant-operator/pull/1030/commits/4bffd5ed4778f30f712b031b006b0c2c5bb6705d, an alternative would be creating a different instance just for the logging that setting or leave it to `fmt.Println`